server/src/main.rs

```diff
@@ -221,10 +226,9 @@ async fn handle_socket(socket: WebSocket, state: Arc<StdMutex<State>>) {
                         }
                     }
                 }
-
                 room.players.write().unwrap().insert(player_id, player);
                 *multiplayer_room.lock().await = Some(room);
-            }
+            },
```

Comment:
This change just clutters the PR. If you had modified lines just above and below, maybe, but not alone like this.